install.sh

```
@@ -980,7 +981,7 @@ fi
 # Use the full path since PATH may not be updated in the current shell session
 # The || true ensures this doesn't fail on older binaries that don't have the setup command
 if [ "$non_interactive" = true ]; then
-  "$INSTALL_DIR/agentuity" setup --non-interactive || true
+  "$INSTALL_DIR/agentuity" setup --non-interactive --setup-token "${SETUP_TOKEN} || true
```

Shellcheck (0.11.0)

[warning] Did you forget to close this double quoted string?

(SC1078)